client/src/featureform/register.py

```diff
@@ -2358,6 +2370,19 @@ def register_training_set(self,
         )
         self.__resources.append(resource)
 
+    def register_model(self, name: str) -> ModelRegistrar:
```

Comment:
I wondered about this, too. Given my frame of reference when starting this work was _"... it'll look a lot like the implementation for `User` ..."_, I was following that pattern. So given `register_user` returns an instance of `UserRegistrar`, I assumed it would be the same for model.

Based on your comment, I'm assuming `register_user` should return an instance of `User` instead, right?